conf/application.conf

```diff
@@ -89,6 +89,7 @@ urls {
   register      = "http://localhost:10030/register-for-crs-and-fatca"
   signOut       = "http://localhost:9025/gg/sign-out"
   lostUTR       = "https://www.gov.uk/find-lost-utr-number"
+  addFI         = "/manage-your-crs-and-fatca-financial-institutions/add"
```

Comment:
If the service name or the /add in the app.routes changes, then this would be out of sync.

Reply:
ah true, shouldve done this:
 ` lazy val addFinancialInstitution: String      = s"$fIManagementUrl/${controllers.addFinancialInstitution.routes.AddFIController.onPageLoad}"`